packages/dx-demo-shell/generate-demos.js

```diff
@@ -222,5 +245,6 @@ const generateDemoRegistry = () => {
 loadThemeNames();
 loadDemosToGenerate();
 generateDemos();
-generateDemoRegistry();
+generateDemoRegistry(DEMOS_REGISTRY_FILE, true);
```

Comment:
Flag is passed to function where it is used once to select another function. Why not just pass function instead?